src/components/charts/publications/editeurs/couts-publication/chart-depenses-estimees.js

```diff
@@ -22,18 +25,76 @@ const Chart = ({ graphFooter, graphComments }) => {
   const chartRef = useRef();
   const intl = useIntl();
   const graphId = 'app.sante-publi.publishers.couts-publication.chart-depenses-estimees';
+  const [publishers, setPublishers] = useState([]);
+  const [publisher, setPublisher] = useState('*');
 
-  // const { observationSnaps, updateDate } = useGlobals();
-  const { updateDate } = useGlobals();
-  // const { data, isLoading, isError } = useGetData(observationSnaps);
-  // const { dataGraph2 } = data;
+  const { observationSnaps, updateDate } = useGlobals();
+  const { data, isLoading, isError } = useGetData(observationSnaps, publisher);
+  const { dataGraphTotal, categoriesYear } = data;
+  const query = getFetchOptions('publishersList', 'health', observationSnaps[0]);
+  const term = {};
+  term[`oa_details.${observationSnaps[0]}.oa_host_type`] = 'publisher';
+  query.query.bool.filter.push({ term });
+  useEffect(() => {
+    Axios.post(ES_API_URL, query, HEADERS).then((response) => {
+      setPublishers(
+        response.data.aggregations.by_publisher.buckets
+          .map((item) => item.key),
+      );
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
 
-  // if (isLoading || !dataGraph2) {
-  //   return <Loader />;
-  // }
-  // if (isError) {
-  //   return <>Error</>;
-  // }
+  if (isLoading || !dataGraphTotal || !categoriesYear) {
+    return <Loader />;
+  }
+  if (isError) {
+    return <>Error</>;
+  }
+  const optionsGraph = getGraphOptions(graphId, intl);
+  optionsGraph.chart.type = 'column';
+  optionsGraph.xAxis = {
+    categories: categoriesYear,
+  };
+  optionsGraph.yAxis = {
+    stackLabels: {
+      enabled: true,
+      // eslint-disable-next-line
+      formatter: function () {
+        // eslint-disable-next-line
+        return cleanBigNumber(this.total).concat(' €');
```

Comment:
façon python ;-)
J'aurais fait :
return `${cleanBigNumber(this.total)} €`;
C'est pareil